src/Components/Assets/AssetsList.tsx

```diff
@@ -1,73 +1,212 @@
+      <div className="md:w-1/2 w-full my-2 mx-auto flex flex-col justify-start items-end">
+        <button
+          onClick={() => setIsScannerActive(false)}
+          className="btn btn-default mb-2"
+        >
+          <i className="fas fa-times mr-2"></i> Close Scanner
+        </button>
+        <QrReader
+          delay={300}
+          onScan={(assetId: any) =>
+            assetId ? navigate(`/assets/${assetId}`) : null
+          }
+          onError={(e: any) =>
+            Notification.Error({
+              msg: e.message,
+            })
+          }
+          style={{ width: "100%" }}
+        />
+        <h2 className="text-center text-lg self-center">Scan Asset QR!</h2>
+      </div>
+    );
+
   return (
-    <div className="px-2 pb-2">
+    <div className="px-4 pb-2">
       <PageTitle title="Assets" hideBack={true} />
+      <div className="md:flex mt-5 space-y-2">
+        <div className="bg-white overflow-hidden shadow rounded-lg flex-1 md:mr-2">
+          <div className="px-4 py-5 sm:p-6">
+            <dl>
+              <dt className="text-sm leading-5 font-medium text-gray-500 truncate">
+                Total Assets
+              </dt>
+              <dd className="mt-4 text-5xl leading-9 font-semibold text-gray-900">
+                {totalCount}
+              </dd>
+            </dl>
+          </div>
+        </div>
+        <div className="flex-1">
+          <InputSearchBox
+            value={qParams.search}
+            search={onSearchSuspects}
+            placeholder="Search by Asset Name"
+            errors=""
+          />
+        </div>
+        <div className="flex-1 flex flex-col justify-start items-end">
+          <div>
+            <div className="flex items-start mb-2">
+              <button
+                className="btn btn-primary-ghost"
+                onClick={() => setShowFilters(true)}
+              >
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  stroke="currentColor"
+                  strokeWidth="2"
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  className="fill-current w-4 h-4 mr-2"
+                >
+                  <line x1="8" y1="6" x2="21" y2="6"></line>
+                  <line x1="8" y1="12" x2="21" y2="12">
+                    {" "}
+                  </line>
+                  <line x1="8" y1="18" x2="21" y2="18">
+                    {" "}
+                  </line>
+                  <line x1="3" y1="6" x2="3.01" y2="6">
+                    {" "}
+                  </line>
+                  <line x1="3" y1="12" x2="3.01" y2="12">
+                    {" "}
+                  </line>
+                  <line x1="3" y1="18" x2="3.01" y2="18">
+                    {" "}
+                  </line>
+                </svg>
```

Comment:
This component is repeating in every resource page. Can we avoid repeatition 😅 
Or we can use [FA Icon](https://fontawesome.com/v5.15/icons/list?style=solid)